nemo/lightning/data.py

```diff
@@ -122,7 +122,7 @@ def add_megatron_sampler(
     global_batch_size: int,
     rampup_batch_size: Optional[List[int]] = None,
     consumed_samples: int = 0,
-    dataloader_type: Literal["single", "cyclic"] = "single",
+    dataloader_type: Literal["single", "cyclic", "batch"] = "single",
```

Comment:
This should also be explained in the doc-string

Reply:
added doc string!